### PR TITLE
Add some more values to beauty=*

### PR DESCRIPTION
**Pull request by Zverik**
Added some frequent non-ambigous values from [taginfo](https://taginfo.openstreetmap.org/keys/?key=beauty#values) and [wiki](https://wiki.openstreetmap.org/wiki/Key:beauty).

### CODE REVIEW

**Comment by tyrasd** [comment]
see my tweaks in 7ef7f8d: what do you think?

**Comment by Zverik** [comment]
Yeah, looks good. `hair_removal` is indeed described in the wiki and has enough usages.

**Comment by Zverik** [comment]
Although I'd argue "Spa" is a wide-understood term that people would search for.

**Comment by tyrasd** [comment]
> Although I'd argue "Spa" is a wide-understood term that people would search for.

I had always associated the term "spa" exclusively with thermal baths (see wikipedia: https://en.wikipedia.org/wiki/Spa) to be honest. Perhaps "Spa Treatment" would be a better description instead of "Hydrotherapy" (but on the other hand, [spa treatment](https://en.wikipedia.org/wiki/Spa#Treatments) can apparently be used for basically any beauty treatment)?
@1ec5 would you mind sharing your opinion as a native speaker?

File: data/fields/beauty.json [hunk 1]
```diff
@@ -4,9 +4,13 @@
     "label": "Services",
     "strings": {
         "options": {
-            "nails": "Nails",
+            "nails": "Manicure / Pedicure",
             "tanning": "Tanning",
-            "cosmetics": "Cosmetics"
+            "cosmetics": "Cosmetics",
+            "spa": "Hydrotherapy",
```
Comment (1ec5) [inline data/fields/beauty.json:10]: I’m not an expert on this industry, but day spas or medspas (medical spas) are very common. They’re distinct from hydrotherapy facilities, which became virtually nonexistent in the U.S. generations ago, and also completely independent of any natural spas.

[](https://commons.wikimedia.org/wiki/File:Philipe's_Day_Spa.jpg)

Comment (tyrasd) [reply data/fields/beauty.json]: thanks, let's use _Day Spa_. I guess the corresponding row of the table on the [wiki](https://wiki.openstreetmap.org/wiki/Key:beauty) could be updated as well. :wink:

Comment (Zverik) [reply data/fields/beauty.json]: Updated the wiki, and the list looks good to me now :)

**Pull request merged**